scripts/seed.py

```diff
@@ -150,6 +153,7 @@ def create_challenge(title, start_date, end_date, host_team):
         published=True,
         enable_forum=True,
         anonymous_leaderboard=False,
+        slug=slug.lower().replace(" ", "-"),
```

Comment:
Can you please add these modifications to slug field below line 142 like
```
slug = '{t}-{y}'.format(t=title, y=year)
slug = slug.lower().replace(" ", "-")
```
and add only slug here.